policy/resolver_test.go

```diff
@@ -572,9 +614,24 @@ framework_maps:
 		require.NoError(t, err)
 		require.NotNil(t, rp)
 		require.Len(t, rp.CollectorJob.ReportingJobs, 9)
-		frameworkJob := rp.CollectorJob.ReportingJobs["sDMg+OsJcXQ="]
+		var frameworkJob *policy.ReportingJob
+		for _, rj := range rp.CollectorJob.ReportingJobs {
+			if rj.QrId == "//test.sth/framework/mondoo-ucf" {
+				frameworkJob = rj
+				break
+			}
+		}
+		require.NotNil(t, frameworkJob)
 		require.Equal(t, frameworkJob.Type, policy.ReportingJob_FRAMEWORK)
-		require.Equal(t, explorer.ScoringSystem_IGNORE_SCORE, frameworkJob.ChildJobs["wzZosRUzLRQ="].Scoring)
```

Comment:
More asking out of curiosity as I have the same kind of tests:

Aren't the checksums stable? Or is this just the easier way in the long run when we change the input?

Reply:
they're not that stable. In this case they seem to have changed. only mrn is stable. so i decided to make it invariant to the checksum